### PR TITLE
update to Test suite latest

### PR DESCRIPTION
**Pull request by bourgeoa**

### CODE REVIEW

**Comment by bourgeoa** [comment]
All tests from the test-suite passed in CI
- web-provider-tests 36/36
- solid-crud-tests 61/61
- web-access-control-tests 67/67

I think that test-suite should be tagged for a nice follow up in NSS CI. latest is not the good solution.

**Review by michielbdejong — APPROVED** [review]
Awesome result, Alain! w00t 🍾

**Comment by michielbdejong** [comment]
> I think that test-suite should be tagged for a nice follow up in NSS CI. latest is not the good solution.

Yes, please use the following tags (I just created them from the 'solidtestsuite' user at Docker hub, let me know if you need access to that yourself):
* [solidtestsuite/webid-provider-tests@v2.0.3](https://hub.docker.com/repository/registry-1.docker.io/solidtestsuite/webid-provider-tests/builds/671036d1-411a-4691-8ee7-c37e6fdb0ecb)
* [solidtestsuite/solid-crud-tests@v2.0.4](https://hub.docker.com/repository/registry-1.docker.io/solidtestsuite/solid-crud-tests/builds/2f40619b-5cc0-4371-8ecd-8d0d0b5cbc0f)
* [solidtestsuite/web-access-control-tests@v2.1.3](https://hub.docker.com/repository/registry-1.docker.io/solidtestsuite/web-access-control-tests/builds/b2b481ee-6993-45f2-96ae-3a832090ef24)

File: test/surface/run-solid-test-suite.sh [hunk 1]
```diff
@@ -45,9 +45,9 @@ teardown || true
 setup
 waitForNss server
 runTests webid-provider-tests latest
```
Comment (bourgeoa) [inline test/surface/run-solid-test-suite.sh:47]:
```suggestion
runTests webid-provider-tests v2.0.3  #latest
```

**Review by bourgeoa — COMMENTED** [review]
replace latest with verified tag version

**Pull request merged**